tools/internal_ci/linux/grpc_xds_bazel_test_in_docker.sh

```diff
@@ -76,3 +77,4 @@ GRPC_VERBOSITY=debug GRPC_TRACE=xds_client,xds_resolver,xds_cluster_manager_lb,c
     --client_cmd='bazel-bin/test/cpp/interop/xds_interop_client --server=xds:///{server_uri} --stats_port={stats_port} --qps={qps} {fail_on_failed_rpc} \
       {rpcs_to_send} \
       {metadata_to_send}'
+!! --xds_v3_support
```

Comment:
This is unfortunate. A single invocation of `run_xds_tests.py` can already take 1+ hr. Doubling that is not going to be fun. We either need separate Kokoro job for v3 tests (I don't see any drawbacks to this?) or push the v2/v3 logic into `run_xds_tests.py` (we already accommodate running multiple clients in parallel for a single `run_xds_test.py` invocation for our staging tests).

Reply:
I agree this approach takes too long.  Will try other approaches.